src/components/Chat/BottomSheetGif.js

```diff
@@ -12,11 +12,10 @@ import FastImage from 'react-native-fast-image';
 import {debounce} from 'lodash';
 
 import {fonts, normalizeFontSize} from '../../utils/fonts';
-import {colors} from '../../utils/colors';
 import {BottomSheet} from '../BottomSheet';
 import MemoIcSearch from '../../assets/icons/Ic_search';
 import dimen from '../../utils/dimen';
-import {SIZES} from '../../utils/theme';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use the new `COLORS` object from the `theme` module instead of the old `colors` module. This is consistent with the changes described in the PR summary.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->